fedimint-server/src/net/api.rs

```diff
@@ -577,11 +577,17 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
         },
         api_endpoint! {
             "auth",
-            async |_config: &ConsensusApi, context, _v: ()| -> () {
+            async |_fedimint: &ConsensusApi, context, _v: ()| -> () {
                 check_auth(context)?;
                 Ok(())
             }
         },
+        api_endpoint! {
+            "modules_json",
```

Comment:
The name seems off. Probably should have a "config" word in it. Should this be maybe `config_json` (to match `config`) and maybe return whole config, but with module configs as json or something? Just thinking aloud.

Reply:
My thinking was that the contents of the client config are already available, so putting them in here would be redundant. Further, it's kind of nice that this can't be accidentally used as a client config which might mess up config signatures somehow?

Also, choose not to just stick all this in the config itself because that is some extra IO that every client would need to do on startup to load the config. Would also be a breaking change that couldn't be applied to release branch backwards-compatibly.

Reply:
`config_json` doesn't make sense because the config is already json. It's just that the `modules` section has consensus hex-encoded values in it.

Reply:
`modules_config_json` then? :D

Reply:
🤝 updated